src/components/ChartContainer/chartcontainer.js

```diff
@@ -0,0 +1,30 @@
+import React from "react";
+import {
+  Container,
+  ChartContainerText,
+  Label,
+  NumberContainer,
+} from "./chartcontainer.styles";
+import { AbbreviatedNumber, Chart } from "components";
+
+export default class ChartContainer extends React.Component {
+  render() {
+    return (
+      <Container>
+        <ChartContainerText>
+          <Label>{this.props.label}</Label>
+          <NumberContainer>
+            <AbbreviatedNumber number={this.props.headerNumber} />
+          </NumberContainer>
+        </ChartContainerText>
+        <Chart
+          data={this.props.data}
+          coin={this.props.coin}
+          chartType={this.props.chartType}
+          dataIndex={1}
+          color={this.props.color}
+        />
+      </Container>
+    );
+  }
+}
```

Comment:
can this be a functional component? if you plan to add state, then it's fine, otherwise consider making it a functional component